compose.py

```diff
@@ -150,41 +151,79 @@ def create_argmap(images):
     # Extract information from upstream build and create map other functions can
     # use
     upstreamDockerImage = images["full"]
+    minDockerImage = images["min"]
```

Comment:
use lower-case underscores style for variables
"upstreamDockerImage" confuses upstream meaning. Use full_docker_image